src/app/pages/WelcomePage/WelcomePage.module.css

```
@@ -0,0 +1,33 @@
+.container {
+  background-image: url('../../components/Welcome/WelcomePageImage.jpg');
+  background-size: cover;
+  height: 100vh;
+  width: 100%;
+}
+
+.icon {
+  position: absolute;
+  top: 1.5rem;
+  left: 0;
+  right: 0;
+  margin: auto;
+}
+
+.form {
+  display: grid;
+  position: absolute;
+  bottom: 2rem;
+  left: 0;
+  right: 0;
+  margin: auto;
+  width: 10rem;
+  gap: 0.5rem;
+}
```

I'd suggest working with a grid here instead of working with a `position: absolute`:

```suggestion
.container {
  display: grid;
  align-content: space-between;
  padding: 1.5em 0 2em;
  background-image: url('../../components/Welcome/WelcomePageImage.jpg');
  background-size: cover;
  height: 100vh;
  width: 100%;
}

.icon {
  margin: auto;
}

.form {
  display: grid;
  margin: auto;
  width: 10rem;
  gap: 0.5rem;
}
```